src/bootstrap-table.js

```diff
@@ -323,6 +315,13 @@ class BootstrapTable {
         html.push('</div>')
         html.push('</th>')
       })
+
+      if (i === 0 && Utils.hasDetailViewIcon(this.options)) {
+        const detailViewTemplate = `<th class="detail" rowspan="${this.options.columns.length}">
+          <div class="fht-cell"></div>
+          </th>`
+        this.options.detailViewHAlign === 'left' ? html.splice(1, 0, detailViewTemplate) : html.push(detailViewTemplate)
```

Comment:
```js
html[this.options.detailViewHAlign === 'right' ? 'push' : 'unshift'](detailViewTemplate)
```

It is better to use `right` because when we use the wrong `value`, it's will use `left` by default.